pimmi/cli/__main__.py

```diff
@@ -146,6 +147,19 @@ def load_cli_parameters():
         'path', type=str, help="Path of the file to be created.")
     parser_create_config.set_defaults(func=create_config)
 
+   # DOWNLOAD command
+    parser_download = subparsers.add_parser(
+        'download-demo', help="Download the dataset files.")
+    parser_download.add_argument(
+        'dataset', help="The dataset to be downloaded. "
+        "You can choose between small_dataset and dataset1. "
```

Comment:
You can use parser.add_argument(...
                    choices=['small_dataset', 'dataset1'],)